### DISCUSSION TITLE
Does places or buildings include residential addresses?

### DISCUSSION THREAD

**Discussion by davidshhh** [OvertureMaps/data#85] [Discussion]
Hi,

I am working on a geospatial project and I'm trying to get all addresses of the US. I am only seeing ~14M addresses in the Overture dataset which is very few given that the US population is > 300M.

Just wanted to make sure I'm not querying the dataset incorrectly or misusing the dataset:

``` sql
SELECT
      addresses[1] as address,
      addresses[1].country as country,
      addresses[1].locality as locality,
      addresses[1].region as region
FROM
       overture_2023_10_19_alpha_0
WHERE
   -- Continental US
       bbox.minX > -124.848974
   AND bbox.maxX < -66.885444
   AND bbox.minY > 24.396308
   AND bbox.maxY < 49.384358

   -- Partition filtering for places
   AND theme = 'places'
   AND type = 'place';

   -- Filter for US
   AND country = 'US'
```

**Comment by Cj-Malone** [comment]
Overture Places are just POIs, as your query shows, some have addresses. But Overture Places will never have residential addresses.

Overture has announced that it will release some addresses, but not the sources or the ETA. (Probably OpenAddresses and Esri. You can get them today.)
